Makefile

```diff
@@ -35,5 +38,12 @@ package-node:
 test-node:
 	@./scripts/test-node.sh
 
+.PHONY: publish-node
+publish-node:
+	@./scripts/publish-node.sh
+
 .PHONY: packages
 packages: package-node package-python
+
+.PHONY: publish
+publish: publish-node publish-python
```

Comment:
@davidgasquez the challenge here with both languages bundled together is that if node fails, python won't run - I wonder if we should somehow swallow a non-zero exit code here or inside `/scripts/publish-node.sh` so it doesn't block the python script from running. What do you think?

Reply:
Heya @djfarrelly, this is a good one to keep in mind! I think we have to take care of it at different levels. 

- If we're deploying it manually in our computers we could use the [`-i` flag in `make`](http://www2.ipp.mpg.de/~dpc/gmake/make_47.html).
- In TravisCI, we can use their [_Deploy To Multiple Providers_ feature](https://docs.travis-ci.com/user/deployment#Deploying-to-Multiple-Providers). This will make it both resilient and faster (parallel deployment). 

With these options, I think we can keep the script cleaner. Would love to hear your thoughts!

Reply:
Good idea @davidgasquez! I've just split up the two in the deploy section of the travis file!